pucauto.py

```diff
@@ -10,6 +12,10 @@
 
 with open("config.json") as config:
     CONFIG = json.load(config)
+    logging.config.dictConfig(CONFIG.get('logging', {
+        # sane defaults here?
+    }))
```

Comment:
I was tempted to put it here.

Anyone who gets exposed to this project is going to only care about changes to logging once they're comfortable messing with the guts of this thing. I say move it here.